README.md

```diff
@@ -1,50 +1,81 @@
 # Making Tax Digital for Income Tax API Changelog
 
-This page contains a log of the latest changes for [Making Tax Digital for Income Tax APIs](https://developer.service.hmrc.gov.uk/api-documentation/docs/api?filter=income-tax-mtd). For information about planned changes to these APIs, see the [Making Tax Digital for Income Tax roadmap](https://developer.service.hmrc.gov.uk/roadmaps/mtd-itsa-vendors-roadmap/).
+This page contains a log of the latest changes
+for [Making Tax Digital for Income Tax APIs](https://developer.service.hmrc.gov.uk/api-documentation/docs/api?filter=income-tax-mtd).
+For information about planned changes to these APIs, see
+the [Making Tax Digital for Income Tax roadmap](https://developer.service.hmrc.gov.uk/roadmaps/mtd-itsa-vendors-roadmap/).
 
 [Get notified about changes to these APIs](notifications/get-notified.md)
 
 ## Support and reporting issues
 
-If you need support with our APIs, or you want to report an issue, please contact our Software Developers Support Team using this [support form](https://developer.service.hmrc.gov.uk/developer/support).
+If you need support with our APIs, or you want to report an issue, please contact our Software Developers Support Team
+using this [support form](https://developer.service.hmrc.gov.uk/developer/support).
 
 ## Mapping APIs to Self Assessment tax return forms
 
-Parameters in some Making Tax Digital for Income Tax APIs map to box numbers in [Self Assessment tax return forms (GOV.UK)](https://www.gov.uk/self-assessment-tax-return-forms). For more information, see [Mapping CSV files](mapping/mapping-csv-files.md).
+Parameters in some Making Tax Digital for Income Tax APIs map to box numbers
+in [Self Assessment tax return forms (GOV.UK)](https://www.gov.uk/self-assessment-tax-return-forms). For more
+information, see [Mapping CSV files](mapping/mapping-csv-files.md).
 
 ## Changelog
 
 **Note:** The date shown is the date that the change was released to Sandbox or Production.
 
 ---
 
+### 23 August 2024
+
+#### Self Assessment Bsas API
+
+The following change is now available in Sandbox and Production for v4.0 and v5.0.
+
+##### Change
+
+For All Endpoints -
+
+- Error RULE_TAX_YEAR_NOT_SUPPORTED is returned when requests are submitted for tax years 2025-26 or later.
```

Comment:
I think we should change "requests" to be more specific- what kind of requests are we talking about here? Maybe update it to "FHL and non-FHL requests".